Take 4: Leadership chapter with 3 new subchapters and landing page (#3136)
alan-turing-institute/the-turing-way: book/website/collaboration/leadership/leadership-features.md

(cl-leadership-features)=
# Important Features of Leadership

Leaders empower others to identify solutions and make their own decisions to address the challenges they face.

> [Monica Cox](https://en.wikipedia.org/wiki/Monica_Cox), an awardee of Purdue University's Faculty Award of Excellent for Leadership, once referred to leadership as "the highest way of activism" ([reference](https://twitter.com/DrMonicaCox/status/1299346141386022913)).

Author: ```suggestion

```{cite}
"A leader is anyone who takes responsibility for finding the potential in people and processes and has the courage to develop that potential. Leadership is not about titles or the corner office. It's about the willingness to step up, put yourself out there, and lean into courage.”
- Brene Brown, [Dare to Lead](https://brenebrown.com/hubs/dare-to-lead/)
```

This statement points to the leadership skill of involving the members of a group (that a leader aims to impact) in decision-making processes and not taking individual decisions for them ([reference](https://www.management-issues.com/opinion/6087/the-leader-as-activist/#:~:text=But%20when%20they%20use%20constructive,in%2Da%2Dlifetime%20act.)).
Leadership is different from operating individually, managing a task or administering a role in isolation.
It is about understanding and enabling the purpose of the project and the process through which people within a team work with each other to reach the overarching goal.

> With this basic understanding of leadership, we discuss some essential leadership features, in contrast to management, in this subchapter.

Author: That's definitely not the intention. These sections are from legacy chapter draft and I might not have done a good job editing this in contract to new content I have created. Let me have another go.

> With this basic understanding of leadership, we discuss some essential leadership features, in contrast to management, in this subchapter.

Author: ```suggestion
With this basic understanding of leadership, we discuss some essential leadership features, in contrast to task management, in this subchapter.
```

## Assigned Leadership

Like a project manager in a team-based research project or a tech company CEO, there are formally-assigned leadership positions.

> A company CEO may undoubtedly have the power to make decisions to ensure that people in their team advance and complete their assigned projects.

Author: ```suggestion
A company CEO have the power to make decisions to provide organisational resources which can impact the overall efficiency and the ability of teams to deliver on organisational goals.
```

> However, such a top-down approach to decision-making is the characteristic of being a manager. It does not necessarily make them a leader. 

Author: ```suggestion
However, they can not ensure that people in their organisation will complete their assigned projects. 
```

> A good leader inspires their team members to make progress in their own right and be leaders themselves, which means leaders are not always managers. Ideally, leaders should not be managers.

Author: oh micromanagers I meant.

> A good leader inspires their team members to make progress in their own right and be leaders themselves, which means leaders are not always managers. Ideally, leaders should not be managers.

Author: ```suggestion
A good leader inspires their team members to make progress in their own right and be leaders themselves, which means leaders should not micromanage their team members and allow spaces for them to grow.
```

> A good leader inspires their team members to make progress in their own right and be leaders themselves, which means leaders should not micromanage their team members and allow spaces for them to grow.

Author: ```suggestion
Individuals within the teams who inspire their team members play huge role in keeping team motivated as they make progress in their own rights, and take on leadership roles themselves.
```

(cl-leadership-features)=
# Important Features of Leadership

> ```{epigraph}

Author: ```suggestion
```{note}
```
